low-level/framework/base/sspl_constants.py

```diff
@@ -107,6 +107,8 @@
 SB_DATA_PATH = DATA_PATH + 'support_bundle'
 SSPL_SB_TMP = f"{SB_DATA_PATH}/sspl_support_bundle_tmp"
 SSPL_SB_REQUEST = f"{SB_DATA_PATH}/sspl-sb-req.txt"
+SB_REQUEST_TIMEOUT = 1500
```

Comment:
why 25 mins? 20 mins should be more than enough, including enclosure logs. have seen more time being taken till 25 mins ?

Reply:
Done